packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/FLTGoogleMapJSONConversions.m

```diff
@@ -113,6 +113,57 @@ GMSMapViewType FGMGetMapViewTypeForPigeonMapType(FGMPlatformMapType type) {
                      markerIds:markerIDs];
 }
 
+FGMPlatformGroundOverlay *FGMGetPigeonGroundOverlay(GMSGroundOverlay *groundOverlay,
+                                                    NSString *overlayId, BOOL isCreatedWithBounds,
+                                                    NSNumber *zoomLevel) {
+  /// Dummy image is used as image is required field of FGMPlatformGroundOverlay and converting
+  /// image back to bitmap image is not currently supported.
+  FGMPlatformBitmap *mockImage =
```

Comment:
This should be called `placeholderImage` rather than `mockImage`, as "mock" generally refers to a specific test construct.